lib/.npmignore

```diff
@@ -5,4 +5,5 @@ android/build
 screenshots
 *-test.js
 stories.js
-__snapshots__
\ No newline at end of file
+__snapshots__
+*.mp4
```

Comment:
can we also ignore `screenshots` while we're at it?

Reply:
It's already ignored :)

Reply:
perfect